sdk/go/common/resource/resource_state.go

```diff
@@ -66,8 +70,8 @@ func NewState(t tokens.Type, urn URN, custom bool, del bool, id ID,
 	external bool, dependencies []URN, initErrors []string, provider string,
 	propertyDependencies map[PropertyKey][]URN, pendingReplacement bool,
 	additionalSecretOutputs []PropertyKey, aliases []URN, timeouts *CustomTimeouts,
-	importID ID, retainOnDelete bool, deletedWith URN) *State {
-
+	importID ID, retainOnDelete bool, deletedWith URN, created *time.Time, updated *time.Time,
+) *State {
```

Comment:
@justinvp @Frassle are we concerned about this breaking some private forks?

Reply:
I don't think so. We've added args to this function in the recent past and haven't gotten any complaints.